nuxt.config.js

```diff
@@ -2,25 +2,19 @@
 export default {
   mode: 'universal',
 
-  head: {
-    titleTemplate: '%s - Iranian Vuejs Meetup'
-  },
+  head: require('./utils/seo/main').mainSEO(),
```

Comment:
Mixing CJS (require) with ESM (export) is not standard. Please use `import ...` syntax.